src/avgScrabbleScore.js

```diff
@@ -7,11 +7,22 @@ let scrabbleScore = require('./scrabbleScore');
  * Use your implementation of scrabbleScore.
  */
 function avgScrabbleScore(words) {
-  // This is your job. :)
+let sum =0 ;
+for (let word of words) {
+  sum+=scrabbleScore(word);
+}
+let avg = parseInt(sum/words.length); //round it down
```

Comment:
Don't round; let the average be the average. The caller can round it down if they want for whatever reason.